src/sentry/db/models/fields/picklefield.py

```diff
@@ -7,7 +7,7 @@
 from sentry.db.models.fields import jsonfield
 from sentry.utils import json
 
-PICKLE_WRITE_JSON = False
+PICKLE_WRITE_JSON = True
 VALIDATE_JSON_SAMPLE_RATE = 0.1
```

Comment:
Would it be worth bumping to to 50 before going full yolo?

Reply:
meh maybe

I'll just put the sample at 100% over the weekend wcgw